setup.py

```diff
@@ -133,7 +133,7 @@ def _parse_requirements_file(file_path):
     "opencv-python<=4.6.0.66",
 ]
 _yolo_integration_deps = [
-    "torchvision>=0.3.0,<0.14",
+    "torchvision>=0.3.0,<=0.15.1",
```

Comment:
We just need to verify torchvision==0.15.1 doesn't break other computer vision flows, like IC. Someone might have already tried it.

Reply:
IC?

Reply:
we would have to verify all IC workflows if we want to bump up `_image_classification_deps` to torchvision==0.15.1 there as well

Reply:
let's keep it as separate to this update. given this is just for the yolo install pathway, we only need to guarantee it works for yolo. if someone has both yolo and ic installed then the dependency resolver should take the lower of the max versions